daml-lf/spec/daml-lf-1.rst

```diff
@@ -2037,6 +2037,38 @@ Note that the fields of struct values are always ordered lexicographically
 by field name, unlike the fields of struct expressions. The field order is
 normalized during evaluation.
 
+Value nesting
+~~~~~~~~~~~~~
+
+  A value is serializable if it lives inside serializable type and its
+  nesting is lower or equal to 100. Formally, testing of a
+  serializable value ``v`` is noted ``|v|`` and is defined recursively
+  on ``v`` as follows (we omit values that do not have serialized type)::
+
+     | LitInt64 | = 0
+     | LitNumeric | = 0
+     | LitBigNumeric | = 0
+     | t | = 0
```

Comment:
What does this `t` refer to? Maybe worth adding as a comment / side condition.

Reply:
`t` is our innovation for text literal, see the line 646. I past here a part of the context where it is defined. 

```
  Expressions
    e ::= x   
       | ...  
       |  ()                                        -- ExpUnit
       |  'True'                                    -- ExpTrue
       |  'False'                                   -- ExpFalse
       |  LitInt64                                  -- ExpLitInt64: 64-bit integer literal
       |  LitNumeric                                -- ExpLitNumeric: Numeric literal
       |  LitBigNumeric                             -- ExpLitBigNumeric: BigNumeric literal
       |  t                                         -- ExpLitText: UTF-8 string literal
       |  LitDate                                   -- ExpLitDate: Date literal
       |  LitTimestamp                              -- ExpLitTimestamp: UTC timestamp literal
       |  LitParty                                  -- ExpLitParty: Party literal
       |  cid                                       -- ExpLitContractId: Contract identifiers
       |  LitRoundingMode                           -- ExpLitRoundingMode: Rounding Mode
       |  F                                         -- ExpBuiltin: Builtin function
       | ...
```

Reply:
Yes, makes sense. I think it would be nice to write it somewhere to the right. 

```
     | t | = 0                  -- string literal
```